src/main/java/com/gluonhq/substrate/util/FileOps.java

```diff
@@ -141,6 +148,44 @@ public static Path copyResource(String resource, Path destination) throws IOExce
         return copyStream(is, destination);
     }
 
+    /**
+     * Copies directory from resources into destination path
+     * @param source
+     * @param target
+     * @throws IOException
+     */
+    public static void copyDirectoryFromResources(String source, final Path target) throws IOException {
+        // https://stackoverflow.com/a/24316335
+        FileSystem fileSystem;
+        try {
+            URI resource = SubstrateDispatcher.class.getResource("").toURI();
+            fileSystem = FileSystems.newFileSystem(resource, Collections.<String, String>emptyMap());
+        } catch(URISyntaxException e) {
+            throw new IOException(e.toString());
+        }
+
+        final Path jarPath = fileSystem.getPath(source);
+
+        Files.walkFileTree(jarPath, new SimpleFileVisitor<Path>() {
+
+            private Path currentTarget;
+
+            @Override
+            public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs) throws IOException {
+                currentTarget = target.resolve(jarPath.relativize(dir).toString());
+                Files.createDirectories(currentTarget);
+                return FileVisitResult.CONTINUE;
+            }
+
+            @Override
+            public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IOException {
+                Files.copy(file, target.resolve(jarPath.relativize(file).toString()), StandardCopyOption.REPLACE_EXISTING);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'StandardCopyOption.REPLACE_EXISTING' due to existing static import 'java.nio.file.StandardCopyOption.REPLACE_EXISTING'](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5574548)